src/workerd/api/gpu/gpu-buffer.c++

```diff
@@ -3,40 +3,50 @@
 //     https://opensource.org/licenses/Apache-2.0
 
 #include "gpu-buffer.h"
-#include <_types/_uint64_t.h>
+#include "workerd/io/io-context.h"
+#include "workerd/jsg/exception.h"
+#include "workerd/jsg/jsg.h"
 
 namespace workerd::api::gpu {
 
-GPUBuffer::GPUBuffer(wgpu::Buffer b, wgpu::BufferDescriptor desc)
-    : buffer_(kj::mv(b)), desc_(kj::mv(desc)) {
+static constexpr char kWebGPUBufferMappingDetachKey[] = "WebGPUBufferMapping";
+
+inline v8::Local<v8::String> V8AtomicString(v8::Isolate *isolate,
+                                            const char *string) {
+  KJ_ASSERT(isolate);
+  if (!string || string[0] == '\0')
+    return v8::String::Empty(isolate);
+  return v8::String::NewFromOneByte(
+             isolate, reinterpret_cast<const uint8_t *>(string),
+             v8::NewStringType::kInternalized, static_cast<int>(strlen(string)))
+      .ToLocalChecked();
+}
```

Comment:
Does `jsg::v8StrFromLatin1(...)` not work for you here?

Reply:
Removed this code.